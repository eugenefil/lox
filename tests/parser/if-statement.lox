if x > 0 {
    x;
}

if x > 0 {
    x;
} else {
    y;
}

if x > 0 {
    x;
} else if y > 0 {
    y;
}
