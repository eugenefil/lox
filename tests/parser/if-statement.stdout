(program
  (if
    (>
      x
      0)
    (block
      x))
  (if
    (>
      x
      0)
    (block
      x)
    (block
      y))
  (if
    (>
      x
      0)
    (block
      x)
    (if
      (>
        y
        0)
      (block
        y))))
